index.html

```diff
@@ -214,7 +214,7 @@ <h2>Clients</h2>
                     <li class="list-inline-item"><button class="btn" id="0-179-12-0_client_tag" client-data-tagbtn="1" onclick="searchForClientTag('0-179-12-0')">0.179.12.0</button></li>
                 </ul>
                 <ul class="list-inline">
-                    <li class="list-inline-item"><button class="btn" id="Stable_client_tag" client-data-tagbtn="1" onclick="searchForClientTag('Stable')">Stable</button></li>
+                    <li class="list-inline-item"><button class="btn active" id="Stable_client_tag" client-data-tagbtn="1" onclick="searchForClientTag('Stable')">Stable</button></li>
```

Comment:
This only visually marks the button as active, the clients don't actually get filtered to only the stable ones.  
Calling `$('#Stable_client_tag').click()` once results in the clients being filtered and the button appearing activated. Or you could do `searchForClientTag('Stable')` and keep the `active` class you added in the html here.